packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -316,7 +316,11 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
             color: game.backgroundColor(),
             child: LayoutBuilder(
               builder: (_, BoxConstraints constraints) {
-                game.onGameResize(constraints.biggest.toVector2());
+                final size = constraints.biggest.toVector2();
+                if (size.isZero()) {
+                  return Container();
```

Comment:
I guess this will quickly flicker a black screen? Could we somehow make the `loadingBuilder` widget run until this is non-zero?

Reply:
Oh yeah, actually we can, i'll do that.